src/supermin-init-prelude.sh

```diff
@@ -16,13 +16,14 @@ LANG=C /sbin/load_policy  -i
 /usr/sbin/dhclient eth0
 
 # set up workdir
-mkdir -p "${workdir:?}"
+mkdir -p "${workdir:?}" /usr/lib/coreos-assembler
 mount -t 9p -o rw,trans=virtio,version=9p2000.L workdir "${workdir}"
+mount -t 9p -o rw,trans=virtio,version=9p2000.L cosa /usr/lib/coreos-assembler
 if [ -L "${workdir}"/src/config ]; then
     mkdir -p "$(readlink "${workdir}"/src/config)"
     mount -t 9p -o rw,trans=virtio,version=9p2000.L source "${workdir}"/src/config
 fi
-mkdir -p "${workdir}"/cache
+mkdir -p "${workdir}"/cache "${workdir}/cache/container-work"
```

Comment:
We could possibly just use the main disk (sda) instead of the cache disk (sdb1) since we are throwing away this content on each run. The one problem there is I think supermin only creates that main disk as an ext2 filesystem so it might not work.

Reply:
Worth an experiment.

Reply:
I changed it to just `/host/containers-storage` and it seemed to work! :+1:

Reply:
we can do in follow-up if it works